src/main/java/stsjorbsmod/cards/cull/Godsbane.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+
+public class Godsbane extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Godsbane.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE_PERCENT = 8;
+    private static final int UPGRADE_DAMAGE_PERCENT = 2;
+
+    public Godsbane() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = DAMAGE_PERCENT;
+        damage = baseDamage = 0;
+    }
+
+    private void recalculateDamage(AbstractMonster m) {
```

Comment:
More curiosity on my part--what's your thinking around having this in a method instead of in the calculateCardDamage() override? In terms of the idea of what it's doing, it seems to me like it exactly is an override for calculateCardDamage(). Just hoping to spark discussion and learn more myself.

Reply:
Hey Fresnoss, Thanks for the reviews. Mainly I was attempting to emulate different cards that I saw that were similar. You're right though, this seems like it should work within the override itself. I will test it again tonight and make sure that it works.

Reply:
Oh yeah, following the existing cards is the way to go.